.github/workflows/longtime.yml

```diff
@@ -0,0 +1,93 @@
+name: "longtime"
```

Comment:
Can we have a better name?

for example, Integration tests

Reply:
ok